src/Components/Web/src/Forms/EditForm.cs

```diff
@@ -75,7 +75,13 @@ protected override void OnParametersSet()
             if ((EditContext == null) == (Model == null))
             {
                 throw new InvalidOperationException($"{nameof(EditForm)} requires a {nameof(Model)} " +
-                    $"parameter, or an {nameof(EditContext)} parameter, but not both.");
+                    $"parameter, or an {nameof(EditContext)} parameter.");
+            }
+            
+            if ((EditContext != null) && (Model != null))
```

Comment:
The comparison above is already matching this case. So this has currently no effect.
https://github.com/aspnet/AspNetCore/blob/67e91e396a2ab4915b7337709892de820346246a/src/Components/Web/src/Forms/EditForm.cs#L75
Is also `true` if both values are not `null`.